sdk/ai/Azure.AI.Projects/src/Generated/AgentsClient.cs

```diff
@@ -5270,6 +5270,7 @@ internal HttpMessage CreateUpdateMessageRequest(string threadId, string messageI
         internal HttpMessage CreateCreateRunRequest(string threadId, RequestContent content, RequestContext context)
         {
             var message = _pipeline.CreateMessage(context, ResponseClassifier200);
+            message.BufferResponse = false;
```

Comment:
@ShivangiReja, if there are other places where the responses contain SSE streams, you will want to set this there as well.

Reply:
Sure, thank you!

Reply:
This file is generated, so we need to move this method into [Custom/Agent/AgentsClient.cs](https://github.com/annelo-msft/azure-sdk-for-net/blob/feature/azure-ai-sdk/agents-sse_stream/sdk/ai/Azure.AI.Projects/src/Custom/Agent/AgentsClient.cs).

Reply:
I moved this to a different place